ParkingLotApiTest/ControllerTest/ParkingLotControllerTest.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using ParkingLotApi;
+using ParkingLotApi.Dtos;
+using ParkingLotApi.Models;
+using ParkingLotApi.Repository;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ParkingLotApiTest.ControllerTest
+{
+    [Collection("ParkingLotTest")]
+    public class ParkingLotControllerTest : TestBase
+    {
+        private readonly ParkingLotContext context;
+        private readonly HttpClient client;
+        private readonly RequestResponseContent content = new RequestResponseContent();
+        public ParkingLotControllerTest(CustomWebApplicationFactory<Startup> factory) : base(factory)
+        {
+            var scope = Factory.Services.CreateScope();
+            var scopeService = scope.ServiceProvider;
+            context = scopeService.GetRequiredService<ParkingLotContext>();
+            client = GetClient();
+        }
+
+        [Fact]
+        public async Task Story1_AC1_Should_add_parkingLot()
+        {
+            // given
+            var parkingLot = new ParkingLot("Lot1", 10, "location1");
+
+            // when
+            var response = await client.PostAsync("/parkinglots", content.GetRequestContent(parkingLot));
+            var responseExistName = await client.PostAsync("/parkinglots", content.GetRequestContent(parkingLot));
+
+            // then
+            Assert.Equal(parkingLot, new ParkingLot(context.ParkingLots.FirstAsync().Result));
+            Assert.Equal(HttpStatusCode.Created, response.StatusCode);
+            Assert.Contains("/parkingLots/Lot1", response.Headers.Location.AbsoluteUri);
+
+            Assert.Equal(HttpStatusCode.BadRequest, responseExistName.StatusCode);
+        }
+
+        [Fact]
+        public async Task Story1_AC2_Should_delete_parkingLot()
```

Comment:
I have a doubt about this type of test. Since the test actually covers two scenarios, the name of this test becomes obscure.